src/Test/Proxy/NginxProxy.php

```diff
@@ -15,7 +15,7 @@ class NginxProxy extends Abstractproxy
 {
     protected $binary = 'nginx';
     protected $configFile;
-    protected $port = 8080;
+    protected $port = 8088;
```

Comment:
yeah, lets solve that separately. but lets not change this value for now then. maybe somebody is using this class in a different context, and then things would break for him.